rplugin/python3/molten/outputbuffer.py

```diff
@@ -218,11 +218,11 @@ def show_virtual_output(self, anchor: Position) -> None:
         )
         self.canvas.present()
 
-    def show_floating_win(self, anchor: Position) -> None:
+    def show_floating_win(self, anchor: Position, offset: int = 0) -> None:
```

Comment:
can we just calculate the offset in here so we don't have to always remember to calculate it before calling this function?

Or at least make it default to calculating it, that way the windows render consistently.